js/views/user_details_dialog_view.js

```diff
@@ -0,0 +1,53 @@
+/* global i18n, Whisper */
+
+// eslint-disable-next-line func-names
+(function() {
+  'use strict';
+
+  window.Whisper = window.Whisper || {};
+
+  Whisper.UserDetailsDialogView = Whisper.View.extend({
+    className: 'loki-dialog modal',
+    initialize({
+      profileName,
+      avatarPath,
+      avatarColor,
+      pubkey,
+      onOk,
+      onStartConversation,
+    }) {
+      this.close = this.close.bind(this);
+
+      this.profileName = profileName;
+      this.pubkey = pubkey;
+      this.avatarPath = avatarPath;
+      this.avatarColor = avatarColor;
+      this.onOk = onOk;
+      this.onStartConversation = onStartConversation;
+
+      this.$el.focus();
+      this.render();
+    },
+    render() {
+      this.dialogView = new Whisper.ReactWrapperView({
+        className: 'edit-profile-dialog',
```

Comment:
Maybe it would be better to give it its own class, so we can make this and the actual "Edit Profile Dialog" look differently if we wanted to? (We don't even need to duplicate styles, can just give the same CSS properties to several classes, e.g. by using comma in the selector).

Reply:
Updated it
Also added the trigger event on the ConversationHeader Avatar click